gef.py

```diff
@@ -11046,6 +11058,14 @@ def maps(self) -> pathlib.Path:
             self._maps = self.root / f"proc/{self.pid}/maps"
         return self._maps
 
+    @property
+    def mode(self) -> RemoteMode:
+        if self.in_qemu_user():
```

Comment:
Can we determine this in the constructor? Do we need to call it every time `.mode` is checked? e.g. in `setup` if the mode is NOT qemu, then we call `in_qemu_user` multiple times.

Reply:
Right, fixed. It does make more sense to be defined in constructor